scripts/revng

```diff
@@ -657,10 +317,10 @@ def main():
         sys.stdout.write("rev.ng version @VERSION@\n")
         return 0
 
-    return run_command(args, unknown_args, post_dash_dash)
+    return run_command(args, unknown_args, post_dash_dash, search_path, command_prefix)
```

Comment:
What is this `command_prefix` thing? Seems like some standalone change has slipped in.
If this is the case, please make a standalone commit.

Reply:
ok i figure it out, there were two global variables, search_path and command prefix.
search_path was the place were programs were searched for, and command_prefix is used to enable stuff like gdb --args. This was needed because now some of the user of those two variables are in another module.

I guess i can split this into two commits and have the one moving to not having those global variables. Not really sure why split them toh. The changes regarding the pipeline require the searchj_path scope reduction

Reply:
If you think you can have a commit that's generally beneficial, create a separate commit.
If this is something conceptually part of another commit, leave it there (and maybe mention it in the commit message). Just make sure that this is not some completely unrelated change slipping in.